rfcs/text/0352-graph-type-system.md

```diff
@@ -1,10 +1,3 @@
-- **Feature Name:** `graph-type-system`
-- **Start Date:** 2022-06-14
-- **RFC PR:** [blockprotocol/blockprotocol#352](https://github.com/blockprotocol/blockprotocol/pull/352)
-- **RFC Discussion:** [blockprotocol/blockprotocol#418](https://github.com/blockprotocol/blockprotocol/discussions/418)
-
-<!-- markdownlint-disable  MD036 -->
-
```

Comment:
This should not be deleted. It is information we want to be present as part of the RFC. Please ask if you unsure before deleting information from a document, it is probably there for a reason.

How does it look with the information present?

Reply:
Hello @CiaranMn, Note has been taken. The reason it was deleted though was that it wasn't being rendered, an error was continuously thrown and prevented the application from launching